crates/conf/src/conf.rs

```diff
@@ -45,6 +45,15 @@ pub struct Camera {
     #[ensure(*rotation_sensitivity > 0., "`rotation_sensitivity` must be greater than 0.0.")]
     pub rotation_sensitivity: f32,
 }
+
+#[derive(Deserialize, Config, Debug, Clone)]
+pub struct AudioConf {
+    pub music_enabled: bool,
+
+    #[is_finite]
+    #[ensure(*music_volume >= 0., "`music_volume` must be greater than 0.0.")]
```

Comment:
* I would enforce that it is strictly larger than `0.0`. Alternatively, get rid of `music_enabled` and interpret `0.0` as disabled.
* Ensure that it is `<= 1.`.